src/client/app/components/csv/ReadingsCSVUploadComponent.tsx

```diff
@@ -3,14 +3,21 @@
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import * as React from 'react';
```

Comment:
In all the files it would be nice to use the VSC command "organize imports" that will sort them in a standard way.